doc/stages/readers.ept.rst

```diff
@@ -44,6 +42,11 @@ resolution
 
     The resulting resolution may not be exactly this value: the minimum possible resolution that is at *least* as precise as the requested resolution will be selected.  Therefore the result may be a bit more precise than requested.
 
+addons
+    A mapping of assignments of the form ``DimensionName: AddonPath``, which assigns dimensions from the specified paths to the named dimensions.  These addon dimensions are created by the `EPT writer`_.  If the dimension names already exist in the EPT `Schema`_ for the given resource, then their values will be overwritten with those from the appropriate addon.
```

Comment:
I guess this should now read the EPT addon writer.

Reply:
:ref:`readers.ept_addon` will get you the top link to the page all the time instead of using HTML links. This will also emit links in the other help content types like man, CHM, and PDF.

Reply:
Addressed in a0f29158c966cafb1e64d886f213d3b4a2ae63e6.